via/static/scripts/video_player/components/test/VideoPlayerApp-test.js

```diff
@@ -463,12 +468,21 @@ describe('VideoPlayerApp', () => {
     const clientConfig = { openSidebar: true };
     const wrapper = createVideoPlayer({ clientConfig });
     const mergedConfig = wrapper.find('HypothesisClient').prop('config');
-    assert.deepEqual(mergedConfig, {
-      openSidebar: true,
-      bucketContainerSelector: '#bucket-container',
-      sideBySide: {
-        mode: 'manual',
-      },
+
+    assert.isTrue(mergedConfig.openSidebar);
+    assert.equal(mergedConfig.bucketContainerSelector, '#bucket-container');
+    assert.equal(mergedConfig.sideBySide.mode, 'manual');
+    assert.isDefined(mergedConfig.sideBySide.isActive);
+  });
+
+  [true, false].forEach(sideBySideEnabled => {
+    it('tells client if side-by-side is active', () => {
+      fakeUseSideBySideLayout.returns(sideBySideEnabled);
+
+      const wrapper = createVideoPlayer();
+      const { sideBySide } = wrapper.find('HypothesisClient').prop('config');
+
+      assert.equal(sideBySideEnabled, sideBySide.isActive());
```

Comment:
The syntax is `assert.equal(actual, expected)`, so the order of arguments should be swapped here, otherwise you'll get confusing error messages if the test fails.

Reply:
If only all testing frameworks got this standardized 😂 

Good catch.